regexes/client/mobile_apps.yml

```diff
@@ -5,6 +5,19 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Naver (https://apps.apple.com/app/id393499958)
+- regex: 'NAVER/(\d+[\.\d]+) CFNetwork'
+  name: 'Naver'
+  version: '$1'
+
+# Naver (https://play.google.com/store/apps/details?id=com.nhn.android.search)
+- regex: 'NAVER\(inapp; search; .+; (\d+[\.\d]+);.+\)'
+  name: 'Naver'
+  version: '$1'
+- regex: 'NAVER\(inapp; search; .+; (\d+[\.\d]+)\)'
+  name: 'Naver'
+  version: '$1'
```

Comment:
can we combine this into one rule?

NAVER\(inapp; search; .+; (\d+[\.\d]+)[;)]